ludwig/encoders/bag_encoders.py

```diff
@@ -49,7 +51,7 @@ def __init__(
             num_fc_layers=0,
```

Comment:
Can you add type hints for the methods inside `bag_encoders`

Reply:
Done.